Makefile

```diff
@@ -141,27 +139,25 @@ endif
 
 ifeq ($(UNICORN_SHARED),yes)
 ifneq ($(filter MINGW%,$(UNAME_S)),)
-LIBRARY = $(BLDIR)/$(LIBNAME).$(EXT)
+LIBRARY = $(LIBNAME).$(EXT)
 else ifneq ($(filter CYGWIN%,$(UNAME_S)),)
-LIBRARY = $(BLDIR)/cyg$(LIBNAME).$(EXT)
```

Comment:
sorry about the ambiguity. by "seperate out", I mean do you want cygwin dll to have the `cyg` prefix and for other dll not to?

I can understand if you do so that cygwin dll can be installed side by side with mingw dlls